src/firefly/js/charts/ui/HistogramOptions.jsx

```diff
@@ -224,6 +255,52 @@ export class HistogramOptions extends React.Component {
     }
 }
 
+//Make the fixed bin layout
+function renderFixedBinSizeOptions(groupKey, histogramParams){
+    return (
+      <div style={{display: 'flex', flexDirection: 'row', padding: '5px 0 15px'}} >
+         <RadioGroupInputField
+            initialState= {{
+                                value: 'bin size selection',
```

Comment:
Here the value should also be `get(histogramParams, 'fixedBinSizeSelection', 'numBins')` - otherwise first time rendering would not be correct. Example: create histogram from drop down 'Charts' menu with bin width option, then open the side options.